go.mod

```diff
@@ -1,3 +1,3 @@
-module github.com/kavenegar/kavenegar-go
+module github.com/mrezagolestan/kavenegar-go
```

Comment:
Please revert the package name to original.

Reply:
Done!